README.md

```diff
@@ -140,3 +140,33 @@ grep-cmso
 # grep for campus topic sent from central forwarder
 grep-cmlo
 ```
+
+
+# Development
+
+If you plan on extending or developing the "platform_config.yml", "configs/", or the setup scripts in "core/", build the Docker image, "Dockerfile-dev", only once using docker-compose -f docker-compose-dev.yml build --no-cache volttron1. Then start the container using docker-compose -f docker-compose-dev.yml up. When you want to make changes to "platform_config.yml", "configs/", or "core/", simply make the changes and then rerun your container. You do not have to rebuild the image every time you make changes to those aforementioned files and folders because they are mounted into the container. The only time you should rebuild the image is when you make changes to the "volttron" source code since that is not mounted to the container but rather baked into the image during the image build.
```

Comment:
I thought you removed Dockerfile-Dev. Could you remove references to Dockerfile-Dev here.

Reply:
Will do.